ksqldb-common/src/main/java/io/confluent/ksql/schema/connect/SqlSchemaFormatter.java

```diff
@@ -161,6 +161,9 @@ public String visitMap(final Schema schema, final String key, final String value
     }
 
     public String visitStruct(final Schema schema, final List<? extends String> fields) {
+      if (fields.isEmpty()) {
+        return "STRUCT< >";
+      }
```

Comment:
Just curious, what does the below collector return with empty fields?

Reply:
`STRUCT<>`